src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,225 @@
+                + "FROM cars_drivers \n"
+                + "JOIN cars "
+                + "ON cars_drivers.cars_id = cars.id\n"
+                + "JOIN manufacturers "
+                + "ON cars.manufacturer_id = manufacturers.id\n"
+                + "WHERE cars_drivers.drivers_id = ? AND cars.is_deleted = FALSE;";
+        List<Car> cars = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement statement
+                        = connection.prepareStatement(query)) {
+            statement.setLong(1, driverId);
+            ResultSet resultSet = statement.executeQuery();
+            if (resultSet.next()) {
+                cars.add(getCarFromResultSet(resultSet));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't get list of cars by driver with id: "
+                        + driverId, e);
+        }
+        return cars;
+    }
+
+    private void formCarsDriversRelation(Car car) {
+        String query = "INSERT INTO cars_drivers (`cars_id`, `drivers_id`) VALUE (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection();
+                   PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setLong(1, car.getId());
+            for (Driver driver: car.getDrivers()) {
+                statement.setLong(2, driver.getId());
+                statement.executeUpdate();
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't create car-driver relation for car "
+                    + car, e);
+        }
+    }
+
+    private void deleteCarsDriversRelation(Long carId) {
+        String query = "DELETE FROM `cars_drivers` WHERE cars_id = ?";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setLong(1, carId);
+            statement.executeUpdate();
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't delete cars relation "
+                    + "from cars_drivers by car id " + carId, e);
+        }
+    }
+
+    private Car getCarFromResultSet(ResultSet resultSet) throws SQLException {
+        Car car = new Car();
+        car.setId(resultSet.getObject("cars_id", Long.class));
+        car.setModel(resultSet.getString("model"));
+        Manufacturer manufacturer = new Manufacturer();
+        manufacturer.setId(resultSet.getObject("manufacturer_id", Long.class));
+        manufacturer.setName(resultSet.getString("manufacturer_name"));
+        manufacturer.setCountry(resultSet.getString("manufacturer_country"));
+        car.setManufacturer(manufacturer);
+        return car;
+    }
+
+    private List<Driver> getDriversForCar(Long carId) {
+        String query = "SELECT id, name, license_number\n"
+                + "FROM drivers\n"

```

Comment:
```suggestion
                + "FROM drivers "

```
`\n` погіршує читаємість коду. Краще прибрати це з усіх query. Можна мість цього прописати пробіл